corehq/apps/app_manager/dbaccessors.py

```diff
@@ -22,14 +22,14 @@ def domain_has_apps(domain):
     return len(results) > 0
 
 
-def get_latest_released_app_doc(domain, app_id, min_version=None):
+def get_latest_released_app_doc(domain, app_id):
```

Comment:
It might be worth attempting to find the commit where it was introduced to see if it was to fix some edge-case bug, but I can't think of what that would be. (Optional extra credit exercise :))